lua/arcademachine_games/sudoku.lua

```diff
@@ -233,7 +235,14 @@ local KANJI do
 end
 
 function GAME:UpdateAttract()
-	if CurTime() > (self.next_attract_tick or 0) then
+	local curtime = CurTime()
+
+	if math.floor(curtime) > (self.last_marquee_tick or 0) then
+		MACHINE:UpdateMarquee()
+		self.last_marquee_tick = curtime
+	end
```

Comment:
The marquee is supposed to be static and DrawMarquee is called automatically on game load so you should only need to call UpdateMarquee if you need to wait for assets to load before drawing it. I don't think your marquee needs any assets so this call doesn't need to be made?

Reply:
The UpdateMarquee() call was intentional. It's being called once per second to achieve the effect that you can see when you load it for testing purposes.
The calling of UpdateMarquee() halts while the player is attached to the arcade machine.